import_export/resources.py

```diff
@@ -667,6 +673,8 @@ def import_row(self, row, instance_loader, using_transactions=True, dry_run=Fals
         """
         skip_diff = self._meta.skip_diff
         row_result = self.get_row_result_class()()
+        if self._meta.store_raw_values:
+            row_result.raw_values = row
```

Comment:
There is a risk here that the user has defined a custom `RowResult` class which doesn't have `raw_values`, in which case this line will cause a crash.  This could be the case with legacy code out there.  It would be safer to use a `hasattr()` check, and there would also have to be a test case for this instance.